src/pages/projectSettings/tabs/NotificationsTab/utils.ts

```diff
@@ -153,7 +41,7 @@ export const generateWebhookSecret = () => {
     "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
 
   for (let i = 0; i < 64; i++) {
-    text += possible.charAt(Math.floor(Math.random() * possible.length));
+    text += possible.charAt(Math.floor(cryptoRandom() * possible.length));
   }
```

Comment:
Have you run into [crypto.getRandomValues()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues)?  We can do the same thing without maintaining a list of possible characters. 
```ts
  const buffer = new Uint8Array(32);
  crypto.getRandomValues(buffer);
  return Array.from(buffer, byte => byte.toString(16).padStart(2, '0')).join('');
```

Reply:
I used `crypto.getRandomValues` in the `cryptoRandom` function, but this is cool! 😲

 My one reservation is that this will generate a token containing only `0-9` and `a-f`. So we'll be missing lowercase `g-z` and uppercase `A-Z`, and I'm not sure if we want to deviate from the current format so much. What do you think?

Reply:
I don't think the characters matter so much. This is more so about ensuring there is enough entropy used to generate the key. You can also increase the radix value used in `toString` to allow for more characters. 
<img width="589" alt="image" src="https://user-images.githubusercontent.com/4605522/228577005-82957f01-2055-46c0-b8cb-a9f470357014.png">